### PR TITLE
Fix transition animation between fragments

### PR DESCRIPTION
**Pull request by pharspher**

### CODE REVIEW

File: app/src/main/java/org/mozilla/focus/fragment/BrowserFragment.java [hunk 1]
```diff
@@ -847,17 +850,36 @@ public boolean onBackPressed() {
...
+     * @param url target url
+     * @param openNewTab whether to load url in a new tab or not
+     * @param onViewReadyCallback callback to notify that web view is ready for showing.
+     */
+    public void loadUrl(@NonNull final String url, boolean openNewTab,
+                        final Runnable onViewReadyCallback) {
+        // In case we call TabsSession#addTab(), which is an async operation calls back in the next
+        // message loop. By posting this runnable we can call back in the same message loop with
+        // TabsContentListener#onFocusChanged(), which is when the view is ready and being attached.
+        final Runnable callback = new Runnable() {
```
Comment (mTwTm) [inline app/src/main/java/org/mozilla/focus/fragment/BrowserFragment.java]: I'm uncertain why we wrap a Runnable with another Runnable here? Is this the intentional delay you mentioned earlier? However this look like nothing is delayed to me.

File: app/src/main/java/org/mozilla/focus/fragment/BrowserFragment.java [hunk 2]
```diff
@@ -120,6 +121,8 @@
 
     private static final int BUNDLE_MAX_SIZE = 300 * 1000; // 300K
 
+    private Handler uiHandler = new Handler(Looper.getMainLooper());
```
Comment (mTwTm) [inline app/src/main/java/org/mozilla/focus/fragment/BrowserFragment.java]: Maybe we can just use ThreadUtils.postToMainThread() if this is a one-time thing?

Comment (pharspher) [reply app/src/main/java/org/mozilla/focus/fragment/BrowserFragment.java]: Ok, I didn't notice there's such a util class, I'll change to use that. Thx!

File: app/src/main/java/org/mozilla/focus/fragment/BrowserFragment.java (hunk 1 above)
Comment (pharspher) [reply app/src/main/java/org/mozilla/focus/fragment/BrowserFragment.java]: Embarrassingly I don't know what I'm doing here, completely nonsense. I'll fix that.

**Pull request merged**